homeassistant/components/fully_kiosk/binary_sensor.py

```diff
@@ -0,0 +1,81 @@
+"""Fully Kiosk Browser sensor."""
+from __future__ import annotations
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+
+SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
+    BinarySensorEntityDescription(key="kioskMode", name="Kiosk Mode"),
+    BinarySensorEntityDescription(
+        key="plugged",
+        name="Plugged In",
+        device_class=BinarySensorDeviceClass.PLUG,
+    ),
+    BinarySensorEntityDescription(
+        key="isDeviceAdmin",
+        name="Device Admin",
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Fully Kiosk Browser sensor."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    sensors = [
+        FullyBinarySensor(coordinator, sensor)
+        for sensor in SENSOR_TYPES
+        if sensor.key in coordinator.data
+    ]
+
+    async_add_entities(sensors, False)
+
+
+class FullyBinarySensor(CoordinatorEntity, BinarySensorEntity):
+    """Representation of a Fully Kiosk Browser binary sensor."""
+
+    def __init__(self, coordinator, sensor):
+        """Initialize the binary sensor."""
+        self.entity_description = sensor
+        self._sensor = sensor.key
+        self.coordinator = coordinator
+
+        self._attr_name = f"{coordinator.data['deviceName']} {sensor.name}"
```

Comment:
With the new entity naming, we don't have to take the device name into account anymore.

```suggestion
```